proto/tendermint/types/types.proto

```diff
@@ -68,17 +68,18 @@ message Header {
   // hashes of block data
   bytes last_commit_hash = 6;  // commit from validators from the last block
   bytes data_hash        = 7;  // transactions
+  int64 data_shares      = 8;
```

Comment:
maybe add in a comment here with a (permanent) link to the LL spec.
Also, while it makes sense to keep related fields closer to each other, the comment `// hashes of block data` implies that this is a hash. I'd suggest adding a newline in between `data_hash` and `data_shares`.

Also, the naming is not clear. IMO, we should aim for parity with the spec where feasible. here `availableDataOriginalSharesUsed` is used in the spec. Either use that name or something that is clearly related. IMO, `num_original_data_shares` would also be clear enough (cc @adlerjohn).

Reply:
I agree naming parity with the specs should be preferred.

Reply:
@liamsi, Should I then revisit all the protos to check their names and order?

Reply:
We should eventually do this bur not necessarily in this PR. Complete parity of all the protos is currently not possible yet as the data structures still differ too much from the ones in the spec anyways. Here it is sufficient to have parity with the name and maybe the position in relation to the other ones in the spec.